styles.css

```diff
@@ -0,0 +1,141 @@
+/* Import */
+@import url('https://fonts.googleapis.com/css2?family=Varta&display=swap');
+
+/* Root */
+:root {
+    --primary-color: #2a9d8f;
+    --primary-dark: #023047;
+    --primary-grey: #eaeaea;
+    --primary-light-color: #e63946;
+    --white: #ffffff;
+}
+
+/* Body */
+body {
+    font-family: Varta;
+}
+
+* {
+    padding: 0;
+    margin: 0;
+}
+/* Navbar */
+.navbar {
+    padding: 1rem;
+    color: var(--primary-dark);
+    display: flex;
+    justify-content: space-around;
+}
+/* logo */
+.logo{
+    width: 5rem;
+}
+.nav-pills {
```

Comment:
this can be removed if empty/unused

Reply:
Done

Reply:
Removed unused and empty code.
And yes, I'll split the single css file into multiple based on components